docs/sdk-writer-guide.md

```diff
@@ -93,9 +93,8 @@ public class TestSdkAvro {
 
     try {
       CarbonWriter writer = CarbonWriter.builder()
-          .withSchema(carbonSchema)
           .outputPath(path)
-          .buildWriterForAvroInput();
+          .buildWriterForAvroInput(new org.apache.avro.Schema.Parser().parse(avroSchema));
```

Comment:
remove these above lines. No need to prepare avro schema now

    // prepare carbon schema from avro schema 
     org.apache.carbondata.sdk.file.Schema carbonSchema =	     org.apache.carbondata.sdk.file.Schema carbonSchema =
             AvroCarbonWriter.getCarbonSchemaFromAvroSchema(avroSchema);	             AvroCarbonWriter.getCarbonSchemaFromAvroSchema(avroSchema);

Reply:
removed